### PR TITLE
Tng 862 prefix tailwind classes

### PR DESCRIPTION
**Pull request by vlapl**
Ready to review.

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -15,13 +16,13 @@ module.exports = {
...
     options: {
       whitelistPatterns: [
-        /^w-/,
-        /^xs:w/,
-        /^sm:w-/,
-        /^md:w-/,
-        /^lg:w-/,
-        /^xl:w-/,
-        /^xxl:w-/,
+        /^ui-w-/,
```
Comment (vlapl) [inline tailwind.config.js:19]: Hier ist meine größte Unsicherheit im PR: soll der Prefix hier auch eingetragen werden?

File: src/components/Quote/style.css [hunk 2]
```diff
@@ -1,28 +1,28 @@
...
-    @apply p-4 flex my-4;
+    @apply ui-p-4 ui-flex ui-my-4;
   }
   .direction--left {
-    @apply flex-row;
+    @apply ui-flex-row;
     .quote {
       &:before {
-        @apply font-awesome;
+        @apply ui-font-awesome;
```
Comment (dbarczewski) [inline src/components/Quote/style.css:10]: I am not sure why, but in the component the quotes are not displayed.

**Review by dbarczewski — CHANGES_REQUESTED** [review]

File: src/components/ProductListItem/__tests__/ProductListItem.spec.tsx [hunk 3]
```diff
@@ -2,12 +2,20 @@ import ProductListItem from "@/components/ProductListItem";
 import { fireEvent, render } from "@testing-library/vue";
 
 describe("components/ProductListItem", () => {
+  const image_url = "original_url";
   const testItem = {
     title: "Product 1",
     description:
-      "The D1R7-TR4P cleaning robot helps you keeping your house free of dirt. It's low height and flexible design means no space will be left out.",
+      "The D1R7-TR4P cleaning robot helps you keeping your house free of dirt. It's low height and ui-flexible design means no space will be left out.",
```
[the remark this answers is not in the document]
Comment (vlapl) [reply src/components/ProductListItem/__tests__/ProductListItem.spec.tsx:9]: This problem and the preciding originate from having tried a search + replace over all files.

File: src/components/Quote/style.css (hunk 2 above)
Comment (vlapl) [reply src/components/Quote/style.css:10]: fixed

File: src/types/components.d.ts [hunk 4]
```diff
@@ -313,7 +313,7 @@ export interface PageProps {
 export class Page extends Component<PageProps> {}
 
 export interface ParagraphProps {
-  size?: "base" | "xs" | "sm" | "md" | "lg";
+  size?: "xs" | "sm" | "base" | "lg" | "xl";
```
Comment (dbarczewski) [inline src/types/components.d.ts:316]: When you change Props please make sure, that the ones you removed are no longer used in the project. 
Often the compiler will tell you this. 
There is a `size="md"` used in the `NewsDetail`-Component. Please change the property there.

**Review by dbarczewski — CHANGES_REQUESTED** [review]
There are a few changes that need to be made. 
In addition there are two tests: 
1. Paragraph.spec.ts
2. Headline.spec.ts
that no longer pass. Please take a look at this.

File: src/documentation/examples/Sections/TeaserSection.slots.tsx [hunk 5]
```diff
@@ -26,14 +26,14 @@ export default class App extends Vue {
           scopedSlots={{
             kicker: txt => (
               <h3
-                class="Headline lg italic include-margin"
+                class="Headline ui-italic ui-include-margin"
```
[the remark this answers is not in the document]
Comment (vlapl) [reply src/documentation/examples/Sections/TeaserSection.slots.tsx:29]: This class only existed as a subclass of _Headline_. The same holds true for include-margin, actually. I removed that too.

**Review by dbarczewski — APPROVED** [review]

**Comment by dbarczewski** [comment]
Well done!

**Pull request merged**